registry/src/store/reducers/adminReducer.js

```diff
@@ -15,8 +15,8 @@ import {
 
 const initialState = {
   error: null,
-  isAdmin: false,
-  message: "",
+  isAdmin: true,
```

Comment:
By default, admin should always have false state. We will change the state when backend verifies that current user is the admin.

Reply:
sure.